src/store/metadata/stub/content/editions/schools-and-nursery-schools-rated_2023-05.json

```diff
@@ -0,0 +1,118 @@
+        "identifier": "2023-05",
+        "title": "Schools and nursery schools rated good or outstanding",
+        "summary": "Percentage of state-funded provided for schools that are rated",
+        "description": "This dataset shows the percentage of state-funded Nursery schools, Primary schools, Secondary schools, Alternative Provision (AP) schools and Special schools that are rated.",
+        "publisher": "https://staging.idpd.uk/publishers/department-for-education",
+        "creator": "https://staging.idpd.uk/publishers/department-for-education",
+        "contact_point": {
+          "name": "ofsted",
+          "email": "mailto:informationrequest@ofsted.gov.uk"
+        },
+        "topics": [
+          "https://staging.idpd.uk/topics/children-education-and-skills"],
+        "frequency": "annual",
+        "keywords": [
+          "state-funded",
+          "inspections-and-outcomes",
+          "subnational",
+          "schools",
+          "county",
+          "region"
+        ],
+        "licence": "http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/",
+        "issued": "2023-05-16T13:36:00Z",
+        "modified": "2023-11-27T15:38:00Z",
+        "spatial_resolution":["E06", "E07", "E08", "E09", "E11"],
+        "spatial_coverage": "E92000001",
+        "temporal_resolution": ["P1Y"],
+        "temporal_coverage": {
+          "start": "2022-01-01T12:00:00Z",
+          "end": "2022-12-31T12:00:00Z"
+        },
+        "versions_url": "https://staging.idpd.uk/datasets/schools-and-nursery-schools-rated/editions/2023-05/versions",
+        "versions": [
+          {
+            "@id": "https://staging.idpd.uk/datasets/schools-and-nursery-schools-rated/editions/2023-05/versions/1",
+            "issued": "2023-05-16T13:36:00Z"
+          }
+        ],
+        "next_release": "2024-05-01T09:30:00Z",
+        "table_schema": {
+          "columns": [
+            {
+              "name": "period",
+              "datatype": "datetime",
+              "titles": "Gregorian Time Interval",
+              "description": "The time period associated with the observation."
+            },
+            {
+              "name": "areacd",
+              "datatype": "string",
+              "titles": "ONS Geography Code",
+              "description": "The statistical geography of the observation being measured."
+            },
+            {
+              "name": "areanm",
+              "datatype": "string",
+              "titles": "Area Name",
+              "description": "The area name associated with the statistical geography."
+            },
+            {
+              "name": "geography",
+              "datatype": "string",
+              "titles": "Geography Type",
+              "description": "The geography type of the area."
+            },
+            {
+              "name": "observation",
+              "datatype": "decimal",
+              "titles": "Percentage of state-funded schools",
```

Comment:
Maybe add 'good or outstanding' to the end of this too.

Reply:
completed